project/src/main/java/com/google/sps/utilities/SpannerClient.java

```diff
@@ -29,23 +29,23 @@
 
 @WebListener
 public class SpannerClient implements ServletContextListener {
-  private static String PROJECT_ID;
+  private static final String PROJECT_ID = DatabaseConstants.PROJECT_ID;
+  private static String DATABASE_ID = DatabaseConstants.DATABASE_ID;
```

Comment:
Can be removed and directly use DatabaseConstants.XX below.

Reply:
Done.